catalog/plugin/catalog-plugin-auditclassifiedaccess/src/test/java/org/codice/alliance/catalog/plugin/auditclassified/AuditClassifiedAccessPluginTest.java

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.catalog.plugin.auditclassified;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.Result;
+import ddf.catalog.data.impl.AttributeImpl;
+import ddf.catalog.data.impl.MetacardImpl;
+import ddf.catalog.data.impl.ResultImpl;
+import ddf.catalog.operation.impl.QueryResponseImpl;
+import ddf.catalog.plugin.PluginExecutionException;
+import ddf.catalog.plugin.StopProcessingException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.codice.alliance.catalog.core.api.types.Security;
+import org.junit.Before;
+import org.junit.Test;
+
+public class AuditClassifiedAccessPluginTest {
+
+  private AuditClassifiedAccessPlugin auditClassifiedAccessPlugin;
+
+  @Before
+  public void setUp() {
+    auditClassifiedAccessPlugin = new AuditClassifiedAccessPlugin();
+  }
+
+  // NOTE: still need to verify the audit message occurs for all the respective tests

```

Comment:
Tip: You can go off of the `SecurityAuditPlugin` as a good example. Just wrap the one line where you do the audit in a `VisibleForTesting` method
https://github.com/codice/ddf/blob/58453c8f6f19053ae17bd3c9e0553301f35256ca/catalog/plugin/catalog-plugin-security-audit/src/main/java/org/codice/ddf/catalog/plugin/security/audit/SecurityAuditPlugin.java#L172
then you can spy on your plugin
https://github.com/codice/ddf/blob/58453c8f6f19053ae17bd3c9e0553301f35256ca/catalog/plugin/catalog-plugin-security-audit/src/test/java/org/codice/ddf/catalog/plugin/security/audit/SecurityAuditPluginTest.java#L49
and then verify how many times it was called and with what parameters
https://github.com/codice/ddf/blob/58453c8f6f19053ae17bd3c9e0553301f35256ca/catalog/plugin/catalog-plugin-security-audit/src/test/java/org/codice/ddf/catalog/plugin/security/audit/SecurityAuditPluginTest.java#L73

Reply:
Ahhh so simple, thank you :tada: